hack/update.sh

```diff
@@ -16,7 +16,7 @@ cp "$ROOT"/generated/1.19/crds/*.concierge.*.yaml "$ROOT/deploy/concierge"
 
 # Make sure we didn't miss any new CRDs.
 crdCount=$(find "$ROOT"/generated/1.19/crds/ -maxdepth 1 -type f -name '*.yaml' | wc -l | tr -d ' ')
-if [[ "$crdCount" != "3" ]]; then
+if [[ "$crdCount" != "4" ]]; then
```

Comment:
I think this check can be removed.

Reply:
This check is to help us as developers make sure that we don't forget to update this script when we add a new CRD.

Reply:
Oh, I see that the script was updated to use glob matching to decide where the CRDs should get copied. Nevermind my above comment... it seems fine to remove this check.

Reply:
Resolved in https://github.com/vmware-tanzu/pinniped/pull/204.